BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -944,6 +944,15 @@
 "SetUpUnlock" = "Set up unlock";
 "SetUpBiometricsOrChooseAPinCodeToQuicklyAccessYourVaultAndAutofillYourLogins" = "Set up biometrics or choose a PIN code to quickly access your vault and autofill your logins.";
 "SetUpLater" = "Set up later";
+"ChooseYourMasterPassword" = "Choose your master password";
+"ChooseYourMasterPasswordMessage" = "Choose a unique and strong password to  keep your information safe.";
+"MasterPasswordRequiredTitle" = "Master password (required)";
+"RetypeMasterPasswordTitle" = "Re-type master password (required)";
+"MasterPasswordHintTitle" = "Master password hint";
+"BitwardenCannotResetLostPasswordMessage" = "Bitwarden cannot reset a lost or forgotten master password.";
+"LearnAboutWaysToPreventAccountLockout" = "Learn about ways to prevent account lockout";
+"WhatMakesAPasswordStrong" = "What makes a password strong";
+"Characters" = "Characters";
```

Comment:
❓ Should this be parametrized (e.g. `%1$@ Characters`)? Would any languages use `Characters 12`?

Reply:
hmm good call, i think this should be parametrized.

Reply:
Not sure if any of the languages we support does that, but "[Fon](https://en.wikipedia.org/wiki/Fon_language)" language invert the order apparently.